modules/controller/rpc-controller.js

```diff
@@ -590,9 +589,8 @@ class RpcController {
                     assertions = await this.dataService.findAssertions(reqNquads);
                 }
                 for (const assertionId of assertions) {
-                    const content = await this.dataService.resolve(assertionId);
-                    if (content) {
-                        const rawNquads = content.nquads ? content.nquads : content.rdf;
```

Comment:
What was content.rdf here?

Reply:
I'd say legacy code that wasn't removed. It's not needed since dataService.resolve will always return nquads.